app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,352 @@
+    <string name="ru_sure_del_group">Êtes-vous sûr de vouloir supprimer ce groupe et tout ce qu'il contient ? Vous ne pouvez pas faire marche arrière.</string>
+    <string name="ru_sure_del_feature">Êtes-vous sûr de vouloir supprimer ces données ?</string>
+    <string name="ru_sure_del_data_point">Êtes-vous sûr de vouloir supprimer cette donnée ?</string>
+    <string name="export_file_name_suffix">%1$s-%2$s-%3$04d%4$02d%5$02d-%6$02d%7$02d%8$02d.csv</string>
+    <string name="backup_file_name_suffix">%1$s-%2$04d%3$02d%4$02d-%5$02d%6$02d%7$02d.db</string>
+    <string name="discrete_values">Ajoutez quelques choix ci-dessous.</string>
+    <string name="feature_name_cannot_be_null">Veuillez ajouter un nom</string>
+    <string name="feature_with_that_name_exists">Vous suivez déjà quelque chose avec ce nom</string>
+    <string name="discrete_value_must_have_name">Veuillez vous assurer que tous vos choix ont un nom</string>
+    <string name="discrete_feature_needs_at_least_two_values">Veuillez ajouter au moins deux réponses à choix multiple</string>
+    <string name="data_points">%1$d Suivi</string>
+    <string name="no_data">Aucune donnée</string>
+    <string name="add_data_point_hint">Entrez une valeur pour : </string>
+    <string name="skip">Passez</string>
+    <string name="exportButton">Exporter</string>
+    <string name="importButton">Importer</string>
+    <string name="export_to">Exporter vers : </string>
+    <string name="import_from">Importer depuis : </string>
+    <string name="import_warning">Avertissement : l'importation est irréversible. Track &amp; Graph tentera de fusionner toute donnée importée avec les données existantes.</string>
+    <string name="graphs_and_statistics"><![CDATA[Graphiques & Statistiques]]></string>
+    <string name="add_a_graph_or_stat">Ajouter un Graphique ou une Statistique</string>
+    <string name="graph_type_label">Type de Graphique : </string>
+    <string name="maximum_sample_duration">Taille de l'échantillon : </string>
+    <string name="select_a_discrete_feature">Sélectionnez un ensemble de données à choix multiple : </string>
+    <string name="no_discrete_features_pie_chart">Impossible de créer un camembert car il n'y a pas donnée suivie à choix multiples.</string>
+    <string name="select_a_feature">Sélectionnez un ensemble de données : </string>
+    <string name="offset">Décalage : </string>
+    <string name="scale">Echelle : </string>
+    <string name="from">De : </string>
+    <string name="to">Jusqu'à : </string>
+    <string name="select_a_value_range">Sélectionnez une plage de valeurs : </string>
+    <string name="graph_name_hint">ex. Exercise : moyenne hebdomadaire glissante</string>
+    <string name="day">Jour</string>
+
+    <string-array name="time_histogram_windows">
+        <item>Horaire</item>
```

Comment:
Google translates the first 3 here as: 

<item> Schedule </item>
<item> Daily </item>
<item> Daily </item>